### ISSUE TITLE
Onboarding using Valet throws "server error"

### ISSUE THREAD

**Issue by DelphiWorlds** [bytefury/crater#24]
I've followed the steps here:

https://docs.craterapp.com/developer-guide.html#installation

..and started onboarding from a Valet URL, in my case: http://crater.test

On the Site URL & Database page, it succeeds in creating tables in the MySQL database, however it does not proceed any further, and indicates "Server Error" in the top right of the screen, with no other information.

**Comment by mohitpanjwani** [comment]
We've just fixed an issue with SQL Modes. Can you do a `git pull` and see that fixes your issue? 

If not, can you update the variable APP_DEBUG=true on `.env` file and mention the error that you're getting?

**Comment by DelphiWorlds** [comment]
Did a pull.. now I get "Database Connection Failed". Setting APP_DEBUG=true does not give me any other information

**Comment by mohitpanjwani** [comment]
Hey we've fixed many issues on 2.0 update. Can you download the new update and check if it works for you?

**Comment by DelphiWorlds** [comment]
Still having the same error, i.e. "Database Connection Failed" :-(

**Comment by mohitpanjwani** [comment]
It seems to work perfectly for me and the rest of the team.. I am using valet too. 

We'll push a small update where it shows the actual error on the installation instead of just "Database connection failed" so we can pinpoint what's happening.

**Comment by rafaelmb** [comment]
How is the error showing on your logs?

**Comment by codernize** [comment]
Hi, been trying to install it but didn't make it trough. 

1. First of all, on onboarding , all checks are ok in first steps (will be back to this after). On the second step I get a warning of some sort that 3 folders don't have (or have) proper permissions. I don't know what to do there, I have 755 . Should it be 777 ? Nothing is mentioned
2.  couldn't get it install db migrations. After it got a 500 error from the xhr request, it had always that error. I found that out to be the migration part for columns that are json. Even though at first the check was made and it was ok, my mysql version doesn't support json column type. 
3. after the previous one, another php error ( https://github.com/laravel/framework/issues/30054 ), but the fix that I've found was to be 'collect_git_information' => false, in flare.php config. I don't have ssh access to server , I'm on shared host.
4. now I am stuck on an access denied for mysql user. I know the password is ok, the db and value is in there , and I get this error (I only assume ) at a later time in the run process, for `select * from `settings` where `option` = profile_complete limit 1` which is pretty simple, right ?
